Dockerfile

```diff
@@ -37,9 +37,19 @@ ENV GIT_VERSION=1:2.39.2-1.1
 
 # Install from backports since renovate requires at least git 2.33.0
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends git=${GIT_VERSION} && \
+    apt-get install -y --no-install-recommends curl git=${GIT_VERSION} && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* && \
     # Configure Git
     git config --global user.email 'bot@renovateapp.com' && \
     git config --global user.name 'Renovate Bot'
+
+# Install Flux CLI
+
+# renovate: datasource=github-releases depName=fluxcd/flux2
+ENV FLUX_VERSION=2.2.3
+
+# Strip 'v' prefix from FLUX_VERSION if it exists
+RUN FLUX_VERSION=$(echo $FLUX_VERSION | sed 's/^v//')
```

Comment:
This can be handled by renovate when updating the environment variable. See here for an exmple: https://github.com/swissgrc/docker-azure-pipelines-git/blob/develop/Dockerfile#L78